src/main/java/io/smartcat/ranger/core/parser/DataGenerator.java

```diff
@@ -63,13 +68,21 @@ public Builder(Map<String, Object> config, String outputValueName) {
          * @return Instance of {@link DataGenerator}.
          */
         public DataGenerator build() {
-            createProxies();
-            parseCompositeValue("", config);
-            return new DataGenerator(proxyValues.get(outputValueName));
+            if (values != null) {
+                createProxies();
+                parseCompositeValue("", values);
```

Comment:
There methods parseCompositeValue and parseSimpleValue are used as void but they have a return type? Whats the intention?